### PR TITLE
fix(roadmap): Fix the date typo

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # KF Serving Roadmap
 ## 2019 - 2020
 
-### v0.4 Performance (ETA: Jan 31, 2019)
+### v0.4 Performance (ETA: Jan 31, 2020)
```
Comment (yuzisun) [inline ROADMAP.md]: @ellis-bigelow you wanna adjust the ETA?

Comment (gaocegege) [reply ROADMAP.md]: I think it will be better to adjust it since it confuses users like me.

**Comment by ellis-bigelow** [comment]
/lgtm
/approve
Let's take a pass at the overall roadmap in our wg meeting this week.

**Pull request merged**